ACadSharp/IO/DXF/DxfStreamWriter/DxfHeaderSectionWriter.cs

```diff
@@ -10,11 +9,8 @@ internal class DxfHeaderSectionWriter : DxfSectionWriterBase
 
 		public CadHeader Header { get { return this._document.Header; } }
 
-		public DxfWriterOptions Options { get; }
-
-		public DxfHeaderSectionWriter(IDxfStreamWriter writer, CadDocument document, CadObjectHolder holder, DxfWriterOptions options) : base(writer, document, holder)
```

Comment:
Options should not be removed from this class.

Reply:
As before. We did not make any changes to this class.